src/solver/conv_asm_implicit_gemm_v4r1_dynamic.cpp

```diff
@@ -272,6 +274,9 @@ bool TunableImplicitGemmV4R1Dynamic::IsValid(const ConvolutionContext& ctx) cons
 
 bool ConvAsmImplicitGemmV4R1DynamicFwd::IsApplicable(const ConvolutionContext& ctx) const
 {
+    if(miopen::IsDisabled(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_FWD_V4R1{}))
+        return false;
+
     const auto device_name = ctx.GetStream().GetDeviceName();
     if(!(StartsWith(device_name, "gfx900") || StartsWith(device_name, "gfx906")))
```

Comment:
[Question] I see that `IsComposableKernelSupportedHardware()` is not used here.
- It is valid for Dynamic iGEMM solvers?
- Does this solver support gfx803/64?
- What about gfx908? Do we have any other Dynamic solver that supports gfx908 and has the same coverage?

Reply:
```IsComposableKernelSupportedHardware()``` is for HIP composable kernel only. 

This is an ASM kernel, and only supports 900/906.

There is another ASM kernel that support 908